kythe/rust/indexer/src/providers.rs

```diff
@@ -91,26 +115,28 @@ impl KzipFileProvider {
 impl FileProvider for KzipFileProvider {
     /// Given a file hash, returns whether the file exists in the kzip.
     fn exists(&mut self, file_hash: &str) -> bool {
-        let name = format!("files/{}", file_hash);
+        // root_name contains a trailing forward-slash, so it's needed in the format
+        // string.
```

Comment:
Consider giving an example here as well, if you think this is worth commenting.

Reply:
Good point, I went ahead and added an example to the comment